### PR TITLE
Add method to get the transform to ECEF coordinates for a tile tree

### PR DESCRIPTION
**Pull request by RBBentley**

### CODE REVIEW

**Comment by RBBentley** [comment]
> What is this function intended to be used for? Is it relevant that individual TileTreeReferences may have a different transform than the TileTree's?

This was requested by the OCP team and they have already tested it - but would be useful for anyone who wishes to know the most accurate transform to place the tile tree on the earth.  Could be internal... I'm not sure if any one would need it outside of OCP.

File: core/frontend/src/tile/TileTree.ts [hunk 1]
```diff
@@ -159,4 +160,28 @@ export abstract class TileTree {
   public accumulateTransformedRange(range: Range3d, matrix: Matrix4d, location: Transform, frustumPlanes?: FrustumPlanes): void {
     this.rootTile.extendRangeForContent(range, matrix, location, frustumPlanes);
   }
+
+  /**
+   * Return the transform from the tile tree's coordinate space to [ECEF] (https://en.wikipedia.org/wiki/ECEF) (Earth Centered Earth Fixed) coordinates.
```
Comment (pmconne) [inline core/frontend/src/tile/TileTree.ts:165]: Link won't display correctly with space between `]` and `(`.

**Review by pmconne — APPROVED** [review]

File: core/frontend/src/tile/TileTree.ts (hunk 1 above)
Comment (RBBentley) [reply core/frontend/src/tile/TileTree.ts:165]: Fixed link.

**Comment by alarkbentley** [comment]
OCP needs this to be able to determine the actual location of the TileTree on earth. So that we then can transform its tiles to an arbitrary coordinate system by overriding the TileTreeReference's computeTransform.

In the normal iModel.js usecase the TileTree is always transformed to the bound iModel's coordinate system. But this is not the case in OCP as we don't always have one single iModel in the scene. ( could be 0, could be multiple )

**Comment by pmconne** [comment]
@Mergifyio backport release/2.16.x

**Pull request merged**